exporter/fileexporter/README.md

```diff
@@ -25,9 +25,9 @@ Exporter supports the following features：
 
 + Support for compressing the telemetry data before exporting.
 
++ Supports for writing into multiple files, where the file path is determined by a resource attribute.
 
 Please note that there is no guarantee that exact field names will remain stable.
-This intended for primarily for debugging Collector without setting up backends.
```

Comment:
Why is this line removed in this PR? Does not seem related to the change.

Reply:
With every new feature added to this exporter, it becomes less and less of a tool for debugging, so the removal of this line is lightly related to the new functionality added in this pr. That was my reason for removing it (after discussing it in the related issue).

But I'm not going to argue about a line in a readme. If you feel strongly about keeping it (or not removing it in this PR because it's not strictly related to it), I'm happy to add it back.

Reply:
> after discussing it in the related issue

Thanks, this is a good point. I'm fine with this. :+1: